doc/user_guide/signal2d.rst

```diff
@@ -50,18 +50,19 @@ Sub-pixel accuracy can be achieved in two ways:
     # combined upsampling and statistical method
     >>> shifts = s.estimate_shift2D(reference="stat", sub_pixel_factor=20)
 
-If you have a large stack of images, you can perform the image alignment step in
-parallel by passing ``parallel=True``. You can control the number of threads used
-with the ``max_workers`` argument. See the :ref:`map documentation <parallel-map-label>`
-for more information.
+If you have a large stack of images, the image alignment is automatically done in
+parallel.
+
+You can control the number of threads used with the ``num_workers`` argument. Or by adjusting
+the scheduler of the :ref:`dask <dask-label>` backend.
```

Comment:
Where is this `dask-label` supposed to point to in the documentation?

Reply:
Let me fix that.  It should point to the information which is now in the "examples/lazy" folder.   I'll move that to user_guide and then add the label back.